ansible/roles/operator-pipeline/templates/openshift/tasks/set-env.yml

```diff
@@ -67,10 +67,10 @@ spec:
             qa)
                 case $ACCESS_TYPE in
                     internal)
-                    PYXIS_URL="https://pyxis.isv.qa.engineering.redhat.com"
+                    PYXIS_URL="https://pyxis.qa.engineering.redhat.com"
                     ;;
                     external)
-                    PYXIS_URL="https://pyxis-isv-qa.api.redhat.com"
+                    PYXIS_URL="pyxis-qa-ext.int.us-west.dc.preprod.paas.redhat.com"
```

Comment:
I guess you unintentionally also removed `https://`

Reply:
Now corrected. Thanks!